utils_linux.go

```diff
@@ -217,7 +218,11 @@ type runner struct {
 }
 
 func (r *runner) run(config *specs.Process) (int, error) {
-	var err error
+	var (
+		err       error
+		handlerCh chan *signalHandler
+		status    int
```

Comment:
nit: I'd rather declare them right there before they are used. It seems more readable this way.

For example:
```go
var handlerCh chan *signalHandler
if !detach {
	handlerCh = newSignalHandler()
}